src/NuGet.Clients/PackageManagement.VisualStudio/Utility/VsHierarchyUtility.cs

```diff
@@ -37,6 +37,28 @@ public static IVsHierarchy ToVsHierarchy(Project project)
             return hierarchy;
         }
 
+        public static VsHierarchyItem GetHierarchyItemForProject(Project project)
+        {
+            Debug.Assert(ThreadHelper.CheckAccess());
+
+            return new VsHierarchyItem(ToVsHierarchy(project));
+        }
+
+        public static string GetProjectId(Project project)
```

Comment:
Its not being used any where, we use it here only to attach these project id to nuget project.